go/runtime/host/sandbox/sandbox.go

```diff
@@ -525,15 +525,25 @@ func (r *sandboxedRuntime) manager() {
 
 // New creates a new runtime provisioner that uses a local process sandbox.
 func New(cfg Config) (host.Provisioner, error) {
+	// Use a default Logger if none was provided.
+	if cfg.Logger == nil {
+		cfg.Logger = logging.GetLogger("runtime/host/sandbox")
+	}
 	// Use a default GetSandboxConfig if none was provided.
 	if cfg.GetSandboxConfig == nil {
 		cfg.GetSandboxConfig = func(hostCfg host.Config, socketPath, runtimeDir string) (process.Config, error) {
+			logWrapper := host.NewRuntimeLogWrapper(
+				cfg.Logger,
+				"runtime_id", hostCfg.Bundle.Manifest.ID.Hex(),
+				"runtime_name", hostCfg.Bundle.Manifest.Name)
 			return process.Config{
 				Path: hostCfg.Bundle.Path,
 				Env: map[string]string{
 					"OASIS_WORKER_HOST": socketPath,
 				},
 				SandboxBinaryPath: cfg.SandboxBinaryPath,
+				Stdout:            logWrapper,
```

Comment:
The PR includes unit tests for the majority of parsing, but it does not cover this integration point (and the one in sgx). Not sure if it's worth adding any tests for this, and where best to add them. I could potentially add checks to [sandbox_test.go](https://github.com/oasisprotocol/oasis-core/blob/1e4123ac5b9293289376aa6191f3a999ca094644/go/runtime/host/sandbox/sandbox_test.go). It would need fixing first, though -- `OASIS_TEST_RUNTIME_HOST_BUNDLE_PATH` is never defined in CI (the older `OASIS_TEST_RUNTIME_HOST_RUNTIME_PATH` is, [here](https://github.com/oasisprotocol/oasis-core/blob/1e4123ac5b9293289376aa6191f3a999ca094644/.buildkite/go/test_and_coverage.sh)), so the test never runs.

Should I add tests anyway? I'm leaning towards "no; manual testing suffices". Is there a better place for these hypothetical tests? It would look a little weird to make the currently very lightweight sandbox_test.go be 80% about logging.

Reply:
Nice catch, can you open an issue for this? The test running script needs to be updated to generate a bundle and set the right env variable.